### ISSUE TITLE
One to many conversation

### ISSUE THREAD

**Issue by adam3278** [freescout-help-desk/freescout#4446]
Hi,
I’m looking for a solution for mass mailing in Freescout. Sometimes, we send control emails (some info from us or data request) to a lot of our customers. Then they can reply and it’s messy. Every message links into original conversation. Replying to the customer email includes other customers messages or the incoming message as reply content when creating the new conversation from the reply. Is there a way to send an email from freescout without feature to link its customer replies into conversation further? I know it’s basic freescout functionality, but sometimes (rare) it’s not intended to be used. Thanks in advance!

**Comment by freescout-help** [comment]
Is this what you are looking for?

![1](https://github.com/user-attachments/assets/b2116829-ad7c-4605-a6fa-b51fd9e3c375)

**Comment by adam3278** [comment]
Exactly! Thank you. This option doesn’t show up, when you try to send mail to multiple bcc.
